src/multiplayer-servers/multiplayer-services/server-protection/gateway-policies.md

```diff
@@ -0,0 +1,55 @@
+# Gateway Policies
+
+Gateway policies provide an additional layer of protection for your game infrastructure by 
+redirecting specific outgoing game server traffic to destinations such as your game backend.
+
+If SteelShield is active, it might also block legitimate connection attempts or responses 
+due to its lack of awareness of your legitimate outgoing traffic.
+
+Implementing Gateway Policies offers an effective solution to prevent both issues.
+
+## Attack Scenario
+
+Without gateway policies, an attacker can spoof a game host's IP address and send malicious requests to your game backend. 
+In response, you or your hosting provider might decide to block the IP address. 
+As a result, your game host, including legitimate game server traffic, 
+would no longer be able to communicate with your game backend.
+
+By redirecting traffic from your game server through the gateway, 
+the traffic originates from gateway IP addresses that are typically unknown to attackers. 
+This makes spoofing a game host's IP address largely ineffective.
+
+## Pre-requisites
+
+In order to protect your game server's traffic, your GameFabric installation requires:
+
+- Protection support enabled, usually indicated by the presence of the Protection/SteelShield navigation item.
+- Sites that support Gateway Policies.
+
+## Gateway Policy
+
+A Gateway Policy specifies a set of destination CIDRs for which outgoing traffic should be redirected.
+
+![create-gateway-policy.png](images/create-gateway-policy.png)
+
+To manage your various game backends or other services you interact with,
+you can create multiple Gateway Policies and assign them to your ArmadaSet, Armada, Formation, and Vessel.
```

Comment:
```suggestion
To manage the services and backends you interact with,
you can create multiple Gateway Policies and assign them to your ArmadaSets, Armadas, Formations, and Vessels.
```

In this case I'd say if we use the singular, use `or`, and if we want to use `and` we have to pluralize the words. Not sure if I'm clear but it makes sense in my head :smile: